core/pta/tests/aes_perf.c

```diff
@@ -0,0 +1,215 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2020, Linaro Limited
+ */
+
+#include <crypto/crypto.h>
+#include <kernel/tee_time.h>
+#include <pta_invoke_tests.h>
+#include <tee_api_defines.h>
+#include <tee_api_types.h>
+#include <trace.h>
+#include <types_ext.h>
+#include <utee_defines.h>
+
+#include "misc.h"
+
+static const uint8_t aes_key[] = {
```

Comment:
Maybe add a comment stating that the keys and IV are the same used in the user-space AES perf TA? (optee_test/ta/aes_perf/ta_aes_perf.c). Although it does not matter much.